README.md

```diff
@@ -33,4 +45,3 @@ Lombok has a lot of useful annotations that give you some of the nice things in
 ### Efficient Primitive Collections
```

Comment:
How does GSCollections perform against Guava (https://github.com/google/guava)? We already use Guava in a few bits and places. (Though it looks like we prefer Scala collections, which don't apply in the context)